src/Beacon.jsx

```diff
@@ -242,6 +236,48 @@ export class Beacon extends Component {
     }
   }
 
+  loadHash(persistent) {
```

Comment:
So every time we mount/update, we check whether the hash is in the database. If it isn't, we set `this.state.hash` so that the beacon will render. If it is, we clear it so the beacon doesn't render (anymore).